tests/backup_helper.go

```diff
@@ -7559,7 +7562,23 @@ func ChangeStorkAdminNamespace(namespace string) (*v1.StorageCluster, error) {
 	if err != nil {
 		return nil, err
 	}
-
+	log.InfoD("Verifying if stork controller configmap is updated with new admin namespace")
+	storkControllerCmUpdate := func() (interface{}, bool, error) {
+		storkControllerConfigMapObject, err := core.Instance().GetConfigMap(storkControllerConfigMap, defaultStorkDeploymentNamespace)
```

Comment:
Is it not px-namespace?

Reply:
This cm will always be deployed in kube-system ns even of portworx is deployed in any other ns
Did I understand the question correctly?